docs/migrating-v1beta1-to-v1.md

```diff
@@ -0,0 +1,80 @@
+<!--
+---
+linkTitle: "Migrating from Tekton v1beta1"
+weight: 4000
+---
+-->
+
+# Migrating From Tekton `v1beta1` to Tekton `v1`
+
+- [Changes to fields](#changes-to-fields)
+- [Replacing `PipelineRunCancelled` with `Cancelled`](#replacing-pipelineruncancelled-status-with-cancelled)
+- [Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`](#upgrading-pipelinerun.timeout-to-pipelinerun.timeouts)
+- [Replacing `Pipeline.Tasks[].Conditions` with `WhenExpressions`](#replacing-pipeline.tasks[].conditions-with-whenexpressions)
+- [Deprecating Resources from Task, TaskRun, Pipeline and PipelineRun](#deprecating-resources-from-task,-taskrun,-pipeline-and-pipelinerun)
+- [Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver](#replacing-taskRef.bundle-and-pipelineRef.bundle-with-bundle-resolver)
+
+
+This document describes the differences between `v1beta1` Tekton entities and their
+`v1` counterparts. It also describes the changed fields and the deprecated fields into v1.
+## Changes to fields
+
+In Tekton `v1`, the following fields have been changed:
+
+| Old field | New field |
+| --------- | ----------|
+| `spec.taskRunSpecs.taskServiceAccountName` | `spec.taskRunSpecs.serviceAccountName` |
+| `spec.taskRunSpecs.taskPodTemplate` | `spec.taskRunSpecs.podTemplate` |
+| `status.taskResults` | `status.results` |
+| `status.pipelineResults` | `status.results` |
+| `taskRun.spec.taskRef.bundle` | removed from `TaskRun` |
+| `pipelineRun.spec.pipelineRef.bundle` | removed from `PipelineRun` |
+
+## Replacing `PipelineRunCancelled` status with `Cancelled`
+`Cancelled` status should be used for replacing the deprecated `PipelineRunCancelled`.
+
+## Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`
```

Comment:
Can you put this in the chart above instead? Also, I would specifically mention that `timeouts.pipeline` is the replacement for `timeout`